<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NFPNest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_resultInfo">
      <property name="title">
       <string>Result info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_stripNb">
         <property name="text">
          <string>Used strip number:XXX</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_utilizationRate">
         <property name="text">
          <string>Utilization rate:XXX</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_fileInfo">
      <property name="title">
       <string>File info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_polygonNb">
         <property name="text">
          <string>Polygon number:XXX</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_totalArea">
         <property name="text">
          <string>Total area:XXX</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_sceneInfo">
      <property name="title">
       <string>Scene info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_stripWidth">
         <property name="text">
          <string>Strip width:XXX</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_stripHeight">
         <property name="text">
          <string>Strip height:XXX</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGraphicsView" name="graphicsView_result"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="action_openFile"/>
    <addaction name="action_saveResult"/>
    <addaction name="action_solve"/>
    <addaction name="action_reset"/>
    <addaction name="action_test"/>
    <addaction name="action_pause"/>
    <addaction name="action_resume"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_openFile">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="action_saveResult">
   <property name="text">
    <string>Save result</string>
   </property>
  </action>
  <action name="action_solve">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="action_reset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="action_test">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="action_pause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="action_resume">
   <property name="text">
    <string>Resume</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
